src/main/java/com/example/TaskApplication/Controller/UserController.java

```diff
@@ -0,0 +1,28 @@
+package com.example.TaskApplication.Controller;
+
+import com.example.TaskApplication.Modal.User;
+import com.example.TaskApplication.Security.JwtUtil;
+import com.example.TaskApplication.Service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class UserController {
+
+  @Autowired
+  UserService userService;
+
+  @PostMapping("/Login")
```

Comment:
Suggestion: Keep Login in small case instead of L in capital. Better we keep it in same case